panoptes_client/subject_set.py

```diff
@@ -34,4 +34,15 @@ def add_subjects(self, subjects):
             json={'subjects': _subjects}
         )
 
+    def add_subject_link(self, subject_id):
+        self.post(
+            '{}/links/subjects'.format(self.id),
+            json={'subjects': [subject_id]}
+        )
+
+    def delete_subject_link(self, subject_id):
```

Comment:
Thinking about it a bit more, I wonder if we should rename `add_subjects` to just `add` and then rename `delete_subject_link` to `remove`. Then using it would be as simple as:

```python
subject_set.add(subject)
subject_set.add([subject1, subject2, subject3])

subject_set.remove(subject)
subject_set.remove([subject1, subject2, subject3])
```